### DISCUSSION TITLE
Hackathon

### DISCUSSION THREAD

**Discussion by mzur** [biigle/core#428] [General]
We had the idea to do a small hackathon on BIIGLE. Here we can collect some topics that could be worked on:

- Replace Mask R-CNN of `biigle/maia` with the implementation of [mmdetection](https://github.com/open-mmlab/mmdetection). Maybe even just use Faster R-CNN. ( biigle/maia#96 )

- Implement instance segmentation for `biigle/maia`. ( biigle/maia#97 )

- Upgrade the Bootstrap CSS framework to a newer version (look and feel of BIIGLE must stay the same!). Or consider to switch to TailwindCSS.

- Try to upgrade Vue ( https://github.com/biigle/core/issues/455 )

- Implement the video timeline as single canvas element. ( biigle/core#417 )

- Anything from the [roadmap](https://github.com/orgs/biigle/projects/4/views/1).

@dlangenk Anything else?

**Comment by dlangenk** [comment]
I need to have a closer look but what comes to my mind right away (order is arbitrary):

- Maia import ( https://github.com/biigle/maia/issues/95 )
- Support for Dino instead of Autoencoder
- Improve the Brush annotation tools
- Implement speech recognition example for selecting favorite labels (1-9 is already trained) ( https://github.com/biigle/core/issues/272 )
